vizro-ai/src/vizro_ai/_vizro_ai.py

```diff
@@ -123,7 +118,7 @@ def plot(
         code_explanation = output_dict.get("code_explanation")
 
         if code_string.startswith("Failed to debug code"):
-            raise DebugFailure(
+            raise Exception(
```

Comment:
Don't you want to continue using `DebugFailure` and import it from the`helper.py`? Is it used elsewhere or just here?